packages/core/src/app/payment/PaymentSubmitButton.tsx

```diff
@@ -68,17 +70,28 @@ const PaymentSubmitButtonText: FunctionComponent<PaymentSubmitButtonTextProps> =
         }
 
         if (methodType === PaymentMethodType.Paypal) {
-            return <TranslatedString id="payment.paypal_continue_action" />;
+            return <TranslatedString
+                data={{ isCheckoutPaymentMethod }}
+                id={
+                    isCheckoutPaymentMethod
+                        ? "payment.paypal_continue_action"
+                        : "payment.paypal_complete_action"
+                }
+            />;
         }
 
         if (methodType === PaymentMethodType.PaypalCredit) {
             return (
                 <TranslatedString
-                    data={{ brandName }}
+                    data={{ brandName, isCheckoutPaymentMethod }}
                     id={
-                        brandName
-                            ? 'payment.continue_with_brand'
-                            : 'payment.paypal_pay_later_continue_action'
+                        isCheckoutPaymentMethod
+                            ? brandName
+                                ? 'payment.continue_with_brand'
+                                : 'payment.paypal_pay_later_continue_action'
+                            : brandName
+                                ? 'payment.complete_with_brand'
+                                : 'payment.paypal_pay_later_complete_action'
```

Comment:
We usually don't use several if conditions like this. Its better to create two separate variables. 

```suggestion
const continueTranslationId = brandName
    ? 'payment.continue_with_brand'
    : 'payment.paypal_pay_later_continue_action'
const completeTranslationId = brandName
    ? 'payment.complete_with_brand'
    : 'payment.paypal_pay_later_complete_action'

....
                        isCheckoutPaymentMethod
                            ? continueTranslationId
                            : completeTranslationId
```